src/client/sandbox/event/message.ts

```diff
@@ -51,7 +50,7 @@ export default class MessageSandbox extends SandboxBase {
     private static _getMessageData (e) {
         const rawData = isMessageEvent(e) ? nativeMethods.messageEventDataGetter.call(e) : e.data;
 
-        return typeof rawData === 'string' ? parseJSON(rawData) : rawData;
+        return typeof rawData === 'string' ? JSON.parse(rawData) : rawData;
```

Comment:
Store JSON's methods in the `nativeMethods` module and use them in the appropriate places.

Reply:
We can't store JSON's methods in the `nativeMethods` module, because it's client-side module. But we also have shared code which uses these methods both on the client and on the server side. So I stored them as global variables in the shared utility.